src/Akka.Persistence.PostgreSql/AkkaContractResolver.cs

```diff
@@ -0,0 +1,27 @@
+﻿using Newtonsoft.Json;
+using Newtonsoft.Json.Serialization;
+using System.Reflection;
+
+namespace Akka.Persistence.PostgreSql
+{
+    // The NewtonSoftJsonSerializer.AkkaContractResolver class has modifier "internal" (PR#2629) and is inaccessible
+    public class AkkaContractResolver : DefaultContractResolver
```

Comment:
What this class is doing?

Reply:
This class, located in Akka.Serialization, has been marked as "internal. See https://github.com/akkadotnet/akka.net/pull/2629/files#diff-18924916afc2076a0470f93f4122f45cL186. So it's not accessible from the plugin.

Either it should be marked as public again but that would need a new package version. Or, as i did, a local version with the same functionality.